libbeat/publisher/pipeline/pipeline.go

```diff
@@ -386,3 +367,36 @@ func (e *waitCloser) dec(n int) {
 func (e *waitCloser) wait() {
 	e.events.Wait()
 }
+
+func makePipelineProcessors(
+	annotations Annotations,
+	processors *processors.Processors,
+	disabled bool,
```

Comment:
Parameter `disabled` is not used in this function.

Reply:
Uh, thanks. It should be passed to `pipelineProcessors`.

Reply:
done